content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -20,99 +20,37 @@ where bottlenecks can be removed to improve overall performance.
-
-## CronJob monitoring
-
-### Kubernetes Job Monitor
-
-With the [Kubernetes Job Monitor](https://github.com/pietervogelaar/kubernetes-job-monitor) dashboard a Cluster Administrator can see which jobs are running and view the status of completed jobs.
-
-### New Relic Kubernetes monitoring integration
-
-[New Relic Kubernetes](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration) integration provides increased visibility into the performance of your Kubernetes environment. New Relic's Kubernetes integration instruments the container orchestration layer by reporting metrics from Kubernetes objects. The integration gives you insight into your Kubernetes nodes, namespaces, deployments, replica sets, pods, and containers.
-
-Marquee capabilities:
-View your data in pre-built dashboards for immediate insight into your Kubernetes environment.
-Create your own custom queries and charts in Insights from automatically reported data.
-Create alert conditions on Kubernetes data.
-Learn more on this [page](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration).
+The resource metrics pipeline provides a limited set of metrics related to
+cluster components such as the Horizontal Pod Autoscaler controller, as well as the `kubectl top` utility.
+These  metrics are collected by the lightweight, short-term, in-memory 
+[metrics-server](https://github.com/kubernetes-incubator/metrics-server) and
+ are exposed via the `metrics.k8s.io` API. 
+
+metrics-server discovers all nodes on the cluster and 
+queries each node's 
+[kubelet](/docs/reference/command-line-tools-reference/kubelet) for CPU and 
+memory usage. The kubelet acts as a bridge between the Kubernetes master and 
+the nodes, managing the pods and containers running on a machine. The kubelet 
+translates each pod into its constituent containers and fetches individual 
+container usage statistics from the container runtime through the container 
+runtime interface. The kubelet fetches this information from the integrated 
+cAdvisor for the legacy Docker integration.  It then exposes the aggregated pod 
+resource usage statistics through the metrics-server Resource Metrics API.
+This API is served at `/metrics/resource/v1beta1` on the kubelet's authenticated and 
+read-only ports. 
+
+## Full metrics pipeline
+
+A full metrics pipeline gives you access to richer metrics. Kubernetes can
+respond to these metrics by  automatically scaling or adapting the cluster
+based on its current state, using mechanisms such as the Horizontal Pod
+Autoscaler. The monitoring pipeline fetches metrics from the kubelet and
+then exposes them to Kubernetes via an adapter by implementing either the
+`custom.metrics.k8s.io` or `external.metrics.k8s.io` API. 
+[Prometheus](https://prometheus.io), a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself.
 
```

Comment:
I wonder if it's worth drafting and using a standard text to suggest that there are also solutions from outside the CNCF, and finding those is up to the reader.
If so, I'd use this here.

Reply:
@sftim I added a generic verbiage

Reply:
My initial response was to agree with @sftim about adding non-specific verbiage, but the more I think about it, the more I think we can cut the language altogether. In effect, we'd be saying:
> Other projects may exist and might help.

It's accurate but not inherently helpful. In this case, I don't think good intentions translate into effective documentation.